homepage/templates/navbar.html

```diff
@@ -1,18 +1,21 @@
 <nav class="navbar navbar-expand-sm navbar-dark">
-	<a class="navbar-brand" href="#">مای‌آکادمی</a>
-	<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
-	        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-		<span class="navbar-toggler-icon"></span>
-	</button>
+    <a class="navbar-brand" href="#">مای‌آکادمی</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
 
-	<div class="collapse navbar-collapse" id="navbarSupportedContent">
-		<ul class="navbar-nav ml-auto">
-			<li class="nav-item active">
-				<a class="nav-link" href="#">خانه</a>
-			</li>
-		</ul>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav ml-auto">
+            <li class="nav-item active">
+                <a class="nav-link" href="#">خانه</a>
+            </li>
+        </ul>
 
-    <button class="btn" type="button">ورود / ثبت‌نام</button>
-
-	</div>
+        {% if user.is_authenticated %}
+            <button class="btn" type="button" onclick="window.location.href ='{% url "logout" %}'">خروج</button>
```

Comment:
Maybe after the complete implementation, we could show some info about the logged-in user in the navbar.

Reply:
Yeah right. Maybe like a username or photo or something.